src/views/blocks/featured-image.php

```diff
@@ -7,12 +7,12 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTCILE_LINK_HERE}
+ * @link http://m.tri.be/1aiy
  *
  * @version 4.7
  *
  */
 
 $event_id = $this->get( 'post_id' );
-?>
-<?php echo tribe_event_featured_image( $event_id, 'full', false );
\ No newline at end of file
+
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_event_featured_image'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 68b46e3aafc8ba116032007f799e8baa</sub>